CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [0.5.25] — Unreleased
+### Fixed
+- Fix metadata extraction to correctly handle integer/floating-point values ([#297](https://github.com/jsvine/pdfplumber/issues/297))
```

Comment:
Given the spec, [as recently noted in the issue thread](https://github.com/jsvine/pdfplumber/issues/297#issuecomment-718862330), perhaps it would be more accurate to remove the word "correctly"?

Reply:
Sure